client/src/modules/reports/generate/cash_report/cash_report.html

```diff
@@ -65,24 +65,11 @@ <h3 class="text-capitalize" translate>REPORT.CASH_REPORT.TITLE</h3>
             </div>
 
             <!-- cashbox selection  -->
-            <div 
-                class="form-group"
-                ng-class="{ 'has-error' : ConfigForm.$submitted && ConfigForm.cashbox.$invalid }">
-                <label class="control-label">
-                    <i class="fa fa-briefcase"></i> <span translate>FORM.SELECT.CASHBOX</span>
-                </label>
-                <select
-                    class="form-control"
-                    name="cashbox"
-                    ng-model="ReportConfigCtrl.reportDetails.account_id"
-                    ng-options="cash.account_id as cash.hrlabel for cash in ReportConfigCtrl.cashboxes track by cash.id"
-                    required>
-                    <option value="" disabled>{{ "FORM.SELECT.CASHBOX" | translate }}</option>
-                </select>
-                <div class="help-block" ng-messages="ConfigForm.cashbox.$error" ng-show="ConfigForm.$submitted">
-                <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
-                </div>
-            </div>
+            <bh-cashbox-select
+              cashboxId = "ReportConfigCtrl.reportDetails.account_id"
```

Comment:
I think this should be `cashbox-id`.  Also, why is this account id?

Reply:
@jniles, the value to be sent is indeed the account Id, but after analysis this report does not work and there is no End to End test to verify the proper functioning of the report / reports / cash_report